services/postService.js

```diff
@@ -0,0 +1,21 @@
+const { postDao } = require("../models");
+const { BaseError } = require("../util/error");
+
+
+const getPostDetail = async ( postId ) => {
+    const { postExists } = await postDao.getPostExists( postId );
+    if ( !+postExists ) throw new BaseError("Post does not exist",400)
```

Comment:
위와 같이 wrapper class를 만들어서 코드를 아래와 같이 수정해보세요~

```suggestion
    const isExist = await postDao.getPostExists( postId );
    
    if ( !isExist ) throw new BaseError("Post does not exist", 400)
    
```